### DISCUSSION TITLE
Programmatically show the modal

### DISCUSSION THREAD

**Discussion by philmetzger** [rainbow-me/rainbowkit#580] [Q&A] [answered]
Hello beautiful people,

I was wondering if there is a way to programmatically show the connect modal?
i.e. instead of using `<ConnectButton />` and getting the user to click that, can I just show the modal instead of that connect button?

**Comment by markdalgleish** [comment, marked as answer]
This is out now in [RainbowKit v0.4.3.](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.4.3)

You can check out the docs here: https://www.rainbowkit.com/docs/modal-hooks

**Reply by philmetzger** [reply]
Awesome thats great news! thanks!

**Reply by 0xfffxfff** [reply]
Great stuff. Thank you for your efforts!
